Upstream-Status: Pending

Subject: rcp: fix to work with large files

When we copy file by rcp command, if the file > 2GB, it will fail.
The cause is that it used incorrect data type on file size in sink() of rcp.

Signed-off-by: Chen Qi <Qi.Chen@windriver.com>
---
 src/rcp.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/rcp.c b/src/rcp.c
index 21f55b6..bafa35f 100644
--- a/src/rcp.c
+++ b/src/rcp.c
@@ -876,9 +876,9 @@ sink (int argc, char *argv[])
   enum
   { YES, NO, DISPLAYED } wrerr;
   BUF *bp;
-  off_t i, j;
+  off_t i, j, size;
   int amt, count, exists, first, mask, mode, ofd, omode;
-  int setimes, size, targisdir, wrerrno;
+  int setimes, targisdir, wrerrno;
   char ch, *cp, *np, *targ, *vect[1], buf[BUFSIZ];
   const char *why;
 
-- 
1.9.1

